CondFormats/CTPPSReadoutObjects/BuildFile.xml

```diff
@@ -1,6 +1,6 @@
 <use name="FWCore/Framework"/>
 <use name="FWCore/MessageLogger"/>
-<use name="FWCore/Concurrency"/>
+<use name="Utilities/Xerces"/>
```

Comment:
@Dr15Jones @ggovi @smuzaffar where is this really used in this package? This questions goes beyond this PR, that can be merged as far as I am concerned

Reply:
never mind, I see it, the code was already using Utilities/Xerces https://cmssdt.cern.ch/lxr/source/CondFormats/CTPPSReadoutObjects/src/CTPPSRPAlignmentCorrectionsMethods.cc#0023